entity-server-api/pom.xml

```diff
@@ -40,5 +40,10 @@
       <artifactId>entity-common-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I know it was my suggestion to use EntityUserException but I think we need a new Exception type in entity-common-api next to MessageCodeException.  What do you think?